.github/workflows/release.yaml

```diff
@@ -41,37 +41,15 @@ jobs:
     runs-on: ubuntu-latest
     if: contains(fromJson('["major", "minor", "patch"]'), inputs.release)
     steps:
-      - name: python
-        uses: actions/setup-python@v4.5.0
-        with:
-          python-version: 3.9
-
       - name: checkout
         uses: actions/checkout@v3.3.0
         with:
           persist-credentials: false
 
-      - name: dependencies
-        uses: py-actions/py-dependency-install@v4.0.0
-
-      - name: user
-        uses: fregante/setup-git-user@v1.1.0
-
-      - name: bump2version
-        run: bump2version ${{ inputs.release }}
-
-      - name: cff
-        run: |
-          sed \
-            's/^date-released: ....-..-../date-released: '`date +%F`'/g' \
-            CITATION.cff >> tmp
-          mv tmp CITATION.cff
-
-      - name: scriv
-        run: scriv collect
-
-      - name: commit
-        run: git commit -am 'Increment ${{ inputs.release }} version'
+      - name: release-bump2version-scriv
```

Comment:
I do not know why but the changes to this file will crash the editor reproducibly; try `moe .github/workflows/release.yaml` with this version of the file and the latest compilation of Moe from the main repository.

Reply:
I assume that (here) line 52 is a very strange edge case of what YAML allows.  When developing the Action, I remember that GitHub's syntax highlighting always complained about the style.  It recommended me to put `${{ inputs.release }}` into quotes as something like a string would have been expected.  The Action itself worked, on the other hand.  So, I guess that the style is just somehow sloppy...

By the way, GitHub does not complain about the style, anymore; I assume they fixed that bug in the meantime.

Reply:
@fox0430

I mostly edit files with Moe, now.  If I am making a somehow urgent change like this and happen to encounter a bug like the one described, I switch to another editor like `ne` (https://github.com/vigna/ne) or `mcedit` (part of the Midnight Commander, `mc`).  Both have YAML syntax highlighting and do not break at all.

To upload changes to GitHub, I copy the file contents from my local machine to the web editor.  As described, it only complained in an earlier version that quotes would have been missing but nothing more happened.

Reply:
@fox0430

GitHub parses syntax with Linguist (https://github.com/github/linguist), `ne` uses just like `mcedit` an automaton description; https://github.com/vigna/ne/blob/master/syntax/yaml.jsf and `cat /usr/share/mc/syntax/yaml.syntax` with `mc` installed.

Maybe Moe's YAML highlighting fails due to the different handling of the three context types? An option might be to refactor it inspired by how `mcedit` does the highlighting.

Reply:
@kevinmatthes 

Thank you. I'll solve this problem in other PRs. First, as a temporary workaround, prevents crashes by #1598 .